models/database.go

```diff
@@ -508,6 +508,10 @@ var databaseSchema = [][]string{
 	37: {
 		`ALTER TABLE agents ALTER COLUMN max_query_log_size TYPE BIGINT`,
 	},
+	38: {
+		`ALTER TABLE artifacts ADD FOREIGN KEY (location_id) REFERENCES backup_locations (id)`,
```

Comment:
How we will handle cases when the user already has some data in artifacts table and it does not satisfy this constraint?

Reply:
let's remove existed artifacts if they not satisfy this constraint

Reply:
done